### PR TITLE
[WIP] Category page updates

### PR DESCRIPTION
**Pull request by jonalport**
I'd like to apologise in advance for the size of this PR - it's way too big! This is largely my fault for not breaking the original front-end user story up into more manageable chunks from the get go.

The objective of this task is to re-skin the category level pages, and integrate with the CMS service to provide the following additional components on the page:
 - Promo image in the header
 - List of feature links in the header
 - Ability to promote up to 2 pieces of content per subcategory

category_page

### CODE REVIEW

**Comment by darokel** [comment]
Eh how comes my commits are here? 👻

File: app/assets/stylesheets/components/page_specific/_link_list.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.link-list__item:before {
+  content: none !important;
+}
+
+.link-list__item:after {
+  content: ", ";
+}
+ 
+.link-list__item:last-child:after {
+  content: "";
```
Comment (benbarnett) [inline app/assets/stylesheets/components/page_specific/_link_list.scss]: You've got `content: none;` above and `content: ""` here. Do these do the same thing? If so could prob choose just one of them.

File: app/assets/stylesheets/components/page_specific/_link_list.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.link-list__item__link {
+  color: $color-true-black !important;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/page_specific/_link_list.scss]: I think avoid `!important` here and just specify the link states so `:link, :visited` etc. That should get specificity right?

File: app/assets/stylesheets/layout/page_specific/_category.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
+// Category level pages
+
+.l-category__heading {
+  margin-top: 3.75rem;
```
Comment (benbarnett) [inline app/assets/stylesheets/layout/page_specific/_category.scss]: Could this use `$baseline-unit`?

File: app/assets/stylesheets/layout/page_specific/_category.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+.l-category__top {
+  border-bottom: 1px solid #edf0f0;
```
Comment (benbarnett) [inline app/assets/stylesheets/layout/page_specific/_category.scss]: Could use a colour variable here

File: app/assets/stylesheets/layout/page_specific/_category.scss [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  background-repeat: no-repeat;
+  background-size: 40% auto;
+  padding-bottom: 60px;
+
+  @include respond-to($mq-s) {
+    background-position: right 70px;
+  }  
+}
+
+p.l-category__top__intro {
```
Comment (benbarnett) [inline app/assets/stylesheets/layout/page_specific/_category.scss]: These seems a little odd. Could this be a child of a container called `.l-category__top__intro`?

File: config/locales/cy.yml [hunk 6]
```diff
@@ -300,6 +300,8 @@ cy:
   categories:
     show:
       view_all: Gweld yr holl ...
+      view_more: View more
```
Comment (benbarnett) [inline config/locales/cy.yml]: Might be safer just to remove these until we get translations – you'll get warnings in dev mode but it'll just pull the english through on prod.

**Comment by jonalport** [comment]
This has been replaced by https://github.com/moneyadviceservice/frontend/pull/1104

**Pull request closed without merging**